gcp/cloud_run_functions/run_followup_response/main.py

```
@@ -107,9 +109,19 @@ def _extract_followup_payload(
 
     Raises:
         pydantic.ValidationError: If the payload fails validation.
     """
-    return FollowupWebServicePayload.model_validate(raw_body)
+    payload = FollowupWebServicePayload.model_validate(raw_body)
+    if payload.phone_number:
+        normalized = normalize_phone_number(payload.phone_number)
+        if not normalized:
+            raise ValueError(
+                f"Could not normalize phone for response {payload.response_id}"
+            )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)